VaticanBAV/et/BAVet123.xml

```diff
@@ -0,0 +1,233 @@
+                            </support>
+                                <extent>
+                                    <measure unit="leaf">76</measure>
+                                    <measure unit="leaf" type="blank">24
+                                        <locus from="1r" to="2v"/>
+                                        <locus target="#67v"/>
+                                        <locus from="72r" to="92v"/>
+                                    </measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>210</height>
+                                        <width>158</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <list>
+                                        <item xml:id="q1" n="A">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="1r" to="2v"/>
+                                        </item>
+                                        <item xml:id="q2" n="1">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="3r" to="10v"/>
+                                        </item>
+                                        <item xml:id="q3" n="2">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="11r" to="18v"/>
+                                        </item>
+                                        <item xml:id="q4" n="3">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="19" to="26v"/>
+                                        </item>
+                                        <item xml:id="q5" n="4">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="27r" to="34v"/>
+                                        </item>
+                                        <item xml:id="q6" n="5">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="35r" to="42v"/>
+                                        </item>
+                                        <item xml:id="q7" n="6">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="43r" to="50v"/>
+                                        </item>
+                                        <item xml:id="q8" n="7">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="51r" to="58v"/>
+                                        </item>
+                                        <item xml:id="q9" n="8">
```

Comment:
The uneven leaf numbers will lead to an error, but probably the catalogue doesn't offer an explanation...

Reply:
Exactly, the catalogue gives no explanation on the internal structure of the quire, here and in many other similar cases. 
I have added a comment on it.